### DISCUSSION TITLE
I am experiencing issues with verifying my domain (teep.africa) on Vercel

### DISCUSSION THREAD

**Discussion by chika-ibegbu** [vercel/community#7430] [Help] [answered]
### Summary

I am experiencing issues with verifying my domain (teep.africa) on Vercel. I have added the required TXT record to my DNS settings in Namecheap, but I still receive the error message stating that another Vercel account is using this domain. Below are the details of my setup:

**Domain:** teep.africa

**TXT Record Details:**
- **Host:** _vercel.teep.africa
- **Value:** vc-domain-verify=teep.africa,d1d1991627ce9a676e9d

**Steps Taken:**
1. Added the TXT record in Namecheap as per the details provided by Vercel.
2. Verified the TXT record using a DNS propagation checker, which shows the correct details.
3. Attempted to add the domain to my Vercel project multiple times.

Despite these steps, I am still unable to verify the domain. Could you please assist me with this issue? If there are any additional steps I need to take, please let me know.

Thank you for your assistance.

### Steps to Reproduce

**Steps Taken:**
1. Added the TXT record in Namecheap as per the details provided by Vercel.
2. Verified the TXT record using a DNS propagation checker, which shows the correct details.
3. Attempted to add the domain to my Vercel project multiple times.

**Comment by pawlean** [comment]
Hi @chika-ibegbu!

I also see that your domain is not resolving. How long ago has it been since you tried this? I wonder if it's just DNS propagating, usually this could take up to 24 to 48 hours.

**Reply by chika-ibegbu** [reply]
Hello @pawlean this has been for about 48hrs, or there about and it says domain is pending verification. how long does domain verification take?

**Reply by pawlean** [reply]
@chika-ibegbu It looks like it's slowly propagating, see [here](https://www.whatsmydns.net/#TXT/_vercel.teep.africa). How does it look for you?

**Reply by chika-ibegbu** [reply]
I can see it on my end also. i will keep you updated

**Reply by chika-ibegbu** [reply]
Hello @pawlean , does it take this long.? I can't see any progress. Is there anything else we need to do

**Reply by chika-ibegbu** [reply]
Hello @pawlean , can you update me on what's going on with the vercel, and possible ways to resolve this.  It's been a week now and I'm loosing customers in my business

**Reply by pawlean** [reply, marked as answer]
Hi @chika-ibegbu!

It seems as though you have a misconfigured nameserver. To fix, login into your Namecheap control panel and remove the `ns1.teep.africa` nameserver.

**Reply by chika-ibegbu** [reply]
Ok, will troubleshoot in namecheap & I'm hoping to give you a positive feedback.

**Reply by chika-ibegbu** [reply]
Hello, problem is solved now. Thanks

[a comment by an automated account is not shown]
